### ISSUE TITLE
Errors when cell reference is prefixed with '$'

### ISSUE THREAD

**Issue by kim3er** [agworld/soroban#3]
To use the example given in the README:

This works:

```
s.C1 = "=SUM(A1, B1:B5, 5) + A1 ^ 3"
```

This doesn't:

```
s.C1 = "=SUM(A1, B1:B5, 5) + $A$1 ^ 3"
```

I'm importing a spreadsheet and would like to either have Soroban ignore the dollars or strip them out. I don't mind doing the work, but I could do with someone pointing me in the right direction. I assume the change would need to go in one of the two 'parse' scripts.

**Comment by kranzky** [comment]
I've just released v0.9.0 which fixes this: https://rubygems.org/gems/soroban/versions/0.9.0

**Comment by kim3er** [comment]
Wow, thank you for resolving this one so quickly.
